### ISSUE TITLE
ATEM mini pro can't connect [BUG]

### ISSUE THREAD

**Issue by Rivenzi** [bitfocus/companion-module-bmd-atem#236]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I need help because my ATEM mini can't connect to companion.

My atem is connected to ethernet and Atem Set up also detects it.
but when I connect the atem in companion (with the correct IP address) I've the status "connecting" and does not move. I tried different versions but nothing changed so I'm a bit lost.

### Expected Behavior

I would like my Atem to be connected to Companion

### Environment (please complete the following information)

```markdown
- OS: windows 10
- Browser: chrome
- Companion Version: v2.4.2
```

**Comment by dx9s** [comment]
I am having same issue, it sometimes connects but only after tinkering with no clear reproducible way to workaround it from not connecting. 
Tried restarting the Mini Pro, restarting Companion, Exporting and Importing followed by restarting Companion, changing IP address to bogus and back, hard code setting ATEM type. 
Have no clue as to why it sometimes connects... MOST of the time it fails to connect.

**Comment by dx9s** [comment]
Okay I just tried these steps:
- Disable connection
- unplug Mini Pro and plug back in
- Enable connection
And it connected immediately? Not sure if this is THE workaround or not.

**Comment by Julusian** [comment]
duplicate of #249
